client/package.json

```diff
@@ -9,6 +9,7 @@
         "@material-ui/icons": "^4.9.1",
         "@material-ui/lab": "^4.0.0-alpha.56",
         "@material-ui/pickers": "^3.3.10",
+        "antalmanac": "file:..",
```

Comment:
```suggestion
```
Please delete this line, run `npm i` again, and then commit/push those changes.

Reply:
Hello! When I delete the line and run `npm install` again, it readds the line. Is that supposed to happen?

Reply:
What version of npm are you running? 
`npm -v`

Reply:
I am running version 8.5.4.